shepherd.js/test/cypress/examples/positioning-strategy.html

```
@@ -0,0 +1,56 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)